packages/components/src/SidePanel/SidePanel.scss

```diff
@@ -22,10 +27,94 @@ $toggle-button-padding: $padding-small - 1;
 		width: $tc-side-panel-width;
 	}
 
-	:global(.btn) svg:global(.tc-svg-icon) {
-		width: $tc-side-panel-icons-size;
-		height: $tc-side-panel-icons-size;
-		margin: 0;
+	&-list {
+		border-right: 2px solid $alto;
+		background-color: $wild-sand;
+
+		.tc-side-panel-list-item.active {
+			background-color: $white;
+			color: $brand-primary;
+			&:hover {
+				:global(.tc-svg-icon) {
+					color: $brand-primary;
+				}
+			}
+			.link {
+				font-weight: bold;
+			}
+		}
+
+		.tc-side-panel-list-item:hover {
+			background-color: $list-item-reverse-hover-color;
+			color: $dove-gray;
+			:global(.tc-svg-icon) {
+				color: $silver;
+			}
+		}
+
+		&.nav-inverse {
+			border-right: none;
+			background-color: $brand-primary;
+
+			.tc-side-panel-list-item:hover,
+			.tc-side-panel-list-item.active {
+				color: $white;
+				:global(.tc-svg-icon) {
+					color: $white;
+				}
+			}
+			.tc-side-panel-list-item:hover {
+				background-color: $list-item-hover-color;
+			}
+
+			.tc-side-panel-list-item.active {
+				background-color: $list-item-active-color;
+			}
+
+			.toggle-btn {
+				&:hover {
+					color: $white;
+				}
+			}
+		}
+	}
+
+	&.large {
+		width: $tc-side-panel-large-width;
+		> ul {
+			width: $tc-side-panel-large-width;
+		}
+
+		.link {
+			:global(.tc-svg-icon) + span {
+				margin-left: $tc-side-panel-large-padding;
+			}
+		}
+
+		:global(.btn.btn-link) > span {
+			width: $tc-side-panel-large-labels-width;
+		}
+
+		&.docked {
+			width: $tc-side-panel-large-docked-width;
+
+			.tc-side-panel-list-item {
+				width: $tc-side-panel-large-docked-width;
+			}
+
+			.nav-pills {
+				.link {
+					padding: $padding-normal $tc-side-panel-large-padding;
+				}
+			}
+
+			.toggle-btn {
+				padding-left: 4px;
```

Comment:
Where does this value come from ?

Reply:
it's for centered align when docked & large display ^^'
I will make a variable :)